src/components/Controls.vue

```diff
@@ -11,23 +25,27 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { mapActions, mapState } from 'vuex'
+import Vue from "vue";
+import { mapActions, mapState } from "vuex";
 
 export default Vue.extend({
   data() {
```

Comment:
@navyhockey56 You can remove all of `data()`, it isn't being used.

Reply:
@sballan Removing either the data or the v-model's cause the page to remove the initial values from the text boxes

Reply:
@navyhockey56 good point